lib/admin_user_index_query.rb

```diff
@@ -106,7 +106,8 @@ def filter_by_search
       if ip = IPAddr.new(params[:filter]) rescue nil
         @query.where('ip_address <<= :ip OR registration_ip_address <<= :ip', ip: ip.to_cidr_s)
       else
-        @query.where('username_lower ILIKE :filter OR email ILIKE :filter', filter: "%#{params[:filter]}%")
+        @query.joins(:primary_email)
+              .where('username_lower ILIKE :filter OR email ILIKE :filter', filter: "%#{params[:filter]}%")
```

Comment:
This query doesn't look right to me. We're joining on the `primary_email` tables but the query is still referencing the `email` column on the users table.